### DISCUSSION TITLE
颱風要來了，大家會想放假嗎

### DISCUSSION THREAD

**Discussion by Jinyuu17** [1101-GS4719-C/Python-Community-GS4719-C#2236] [General]
上次山陀兒颱風放了兩天的颱風假，老實說想當的爽，終於可以睡到自然醒了。這次的颱風根據報導路經跟上次差不多，但風速雨量又更強了，大家會希望放颱風假來讀期中考嗎？

有研究顯示，市長放對颱風假會被大家稱作「神」，增加民眾的好感度；該放假而不放假對支持度的下降明顯可見；但不該放假卻放假反而對支持度沒有影響。綜上所述，我們善政市長應該讓我們放假了吧，放颱風假的CP值超級高的，放對了我第一個叫他「張神」。

**Comment by yuan3x2** [comment]
沒有課，放不放假其實無所謂，但颱風天下雨真的很麻煩，出門吃飯不便，外送也不能叫。因此，我還是偏愛無風無雨的颱風假，既能輕鬆自在，又無後顧之憂！

**Comment by gigitilto** [comment]
我認為放假是很好的，但也不太好，因為好就是好在放假，不好就是不好在不太能出門，這會導致一些影響，例如出門鞋子會濕掉，很麻煩，而且也不能去想去的地方，窩在家裡有時候就會無聊，當然也可能不會無聊，因為有網路可以使用。

**Comment by Boying12** [comment]
颱風期間也可能造成交通不便，外出什麼的都很麻煩。尤其是在颱風來臨時，風雨可能會非常猛烈，這使得行駛在路上的車輛不僅速度減緩，甚至可能會遇到封路的情況。因此，大家在這段時間內往往會選擇待在家中，以確保自身的安全。不過，出於其他考量，我並不希望放假。許多機關和單位會因此關門，這將導致許多緊急事務的處理有所延宕。比如，醫療機構可能無法正常運作，對於有需要的人來說，這是一個潛在的風險。同樣地，政府機構若無法正常運行，可能會影響到災後救援和物資調配的速度，進而影響到民眾的生活。因此，我認為在颱風期間，應該盡量保持正常的運作，讓大家在這樣的特殊情況下，仍然能夠獲得必要的支持和服務，這樣才能更好地應對颱風帶來的挑戰。

**Comment by Tchang323** [comment]
雖然已經過了
但還是回一下XD
不想放颱風假，因為我禮拜四只有一堂羽球課，很想打羽球

**Comment by ShuliANN** [comment]
颱風來的時候，確實很多人會期待放假，畢竟能放颱風假通常意味著可以有時間休息或避開壞天氣。不過，從經濟角度看，放颱風假確實對GDP會有影響，尤其是全國性的停工停課，經濟損失可能相當可觀。根據以往數據，一天的颱風假可能會造成數十億元的經濟損失，對許多產業都有影響。

**Comment by ShuliANN** [comment]
而且這學期的颱風假都剛好集中在星期四，我自己私心想上網球課，被放掉好多堂，都沒有機會好好學網球，所以不太想放。不過，安全始終是最重要的考量，如果天氣惡劣或有明顯的安全隱患，當然放假還是有它的必要性。

**Comment by yue083139** [comment]
這學期都是禮拜四被放到颱風假，第一次放的時候剛好放到一科的期中考害我隔周考試擠在一起，不過有一說一，颱風假感覺還是有必要的，颱風天風大雨大的一想到要出門上課就頭痛，如果風雨是真的大的話還是依照停課標準去停班課比較好。

**Comment by lai1008** [comment]
這學期剛好都是禮拜四被放到颱風假，第二次放的時候剛好放到一科的期中考害我運動會當天還要考試，不過有一說一，颱風假還是有必要的，如果颱風天風大雨大的一想到要出門上課就頭痛，若風雨是真的大的話還是依照停課標準去停班課比較好。

**Comment by Elaine4488** [comment]
如果沒有風雨，我其實更傾向照常上班或上課。以前放假只會考慮要不要去上課，但這次剛好碰到打工，因為放假沒有薪水，難免有點失落。而且，放假還會打亂許多行程的安排，所有事情都得延後或改期處理，覺得有些麻煩。

**Comment by kzx-creat** [comment]
一律支持放假! 風雨大的話我是真很不想出門騎摩托車，阿如果是沒風沒雨的颱風假，我都當賽到 :)

**Comment by 113607514** [comment]
上次山陀兒颱風來時放了兩天的颱風假，老實說，那段時間真的是讓我過得很愜意，終於可以睡到自然醒。這次的颱風雖然路徑和上次差不多，但風速和雨量都更強了。大家會希望放颱風假來讀期中考嗎？

有研究指出，市長如果能準確地放颱風假，民眾會稱讚他是「神」，並大幅提升支持度；如果應該放假卻沒放，支持度就會明顯下降；而不該放假卻放的話，則對支持度影響不大。總結來看，我們的聰明市長應該趕快決定放假，畢竟放對了假，CP值真的是超高！

**Comment by wcyeee** [comment]
感覺放颱風假除了學生之外其他人的壓力都會很大...市長或行政人員要猜到底需不需要放，雖然有資訊可以幫助判斷，但也不是百分之百準確，放對了就沒事，但放錯的話就要被民眾罵。然後上班族們也會很擔心因為颱風放的是颱風假... 然後對大家也都會有一定的危險，許多作物或是牲畜也都會被影響，所以我覺得放颱風假雖然快樂，但還是希望不要有颱風...

**Comment by Mminniehuang** [comment]
鄰近期末考又回想到期中考週經歷的颱風，我覺得就上次的經驗，期中考延後使得本來分散的考試科目全部集中在一起真的太痛苦了，但是出門覓食看到滿地的樹枝又覺得放假很有必要。
常常看到別人評論說只有學生才會想放假，我還是好好珍惜現在的狀態，也希望颱風天大家都平安。

**Comment by poyen0806** [comment]
這次颱風讓我印象很深刻，租屋處停電了，讓生活有點措手不及，我的手機沒電！！！！！

最後只好跟學校請了心理假回家避難，雖然因此錯過了一堂體育課，但安全最重要，這樣的選擇也是必要的，絕對不是剛好不想上體育課。

希望颱風過後，大家都能平安無事！

**Comment by BOSENCHANG2004** [comment]
這次颱風我最印象深刻的是我和對象看著外面風雨暫時停下來就直接衝往桃園市區去吃兩餐，結果隔天正常上班上課，原本還想說不用上課。這次說走就走的經驗讓我印象深刻。

**Comment by lichilun** [comment]
我當然想放啊，但問題是還是要看學校會不會放假，記得有幾次放颱風假，但其實沒有什麼雨和風，當下真的有賺到的感覺。

**Comment by Piyenparty** [comment]
感覺這個問題可以在許多不同的焦點上做討論，是要討論對經濟的影響，對學生的影響，還是對安全層面的影響etc。對經濟而言，放颱風假肯定是不樂見的，雖然我不是這方面的專家，但在我印象中放一天颱風假會虧超多錢，但具體是多少錢我也不知道，正如我一開始所說的，我不是這方面的專家。對學生影響肯定有好有壞，好的是可以在緊繃的情緒中得到解脫，但沒上到課真的很虧；而對於安全而言，減少通勤確實能減低颱風帶來的災情，畢竟雨中的交通真的很虧賊
